.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -18,12 +30,18 @@ jobs:
       run: |
         set -eu
         since_time=$(date -u -d '24 hours ago' '+%a, %d %b %Y %H:%M:%S GMT')
-        code=$(curl \
-            -o nvd.zip \
-            -w "%{http_code}" \
-            -H "If-Modified-Since: $since_time" \
-            https://definitions.stackrox.io/v4/nvd/nvd.zip)
-
+        if [ "$JOB" = "nvd-api" ]; then
+          url="https://definitions.stackrox.io/v4/nvd/nvd-api.zip"
+        else
+          url="https://definitions.stackrox.io/v4/nvd/nvd-feeds.zip"
+        fi
+        
+        curl \
+          -o nvd.zip \
+          -w "%{http_code}" \
+          -H "If-Modified-Since: $since_time" \
+          $url \
+        | grep 200
```

Comment:
```suggestion
        code=$(curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            "$url")
```